blocks/header/header.js

```diff
@@ -324,6 +407,93 @@ const decorateHamburgerPanel = (fragment, block) => {
   block.append(sidePanelDiv);
 };
 
+/**
```

Comment:
Probably this can go in  `scripts/utils.js` as may be needed at other places.

Reply:
done, moved it to utils